app/controllers/photos_controller.rb

```diff
@@ -8,6 +8,7 @@ def find_album
 
   def find_photo
     @photo = Photo.find(params[:id])
+    @photo.view += 1;
```

Comment:
I think views might be a bit more understandable than view- view sounds a bit like a method name, so it's not as intuitive. Does this also save to the database? Alternatively you could use @photo.increment!(:views) :)